packages/react-native/Libraries/AppDelegate/RCTAppSetupUtils.mm

```diff
@@ -124,16 +124,6 @@ void RCTAppSetupPrepareApp(UIApplication *application, BOOL turboModuleEnabled)
   // Necessary to allow NativeModules to lookup TurboModules
   [bridge setRCTTurboModuleRegistry:turboModuleManager];
 
-#if RCT_DEV
-  /**
-   * Instantiating DevMenu has the side-effect of registering
-   * shortcuts for CMD + d, CMD + i,  and CMD + n via RCTDevMenu.
-   * Therefore, when TurboModules are enabled, we must manually create this
-   * NativeModule.
-   */
-  [turboModuleManager moduleForName:"RCTDevMenu"];
-#endif // end RCT_DEV
-
```

Comment:
Because this is inside `RCT_REMOVE_LEGACY_ARCH`, I guess this code is technically frozen?